README.md

```diff
@@ -1 +1,37 @@
 # Gemstock-Mk2
+
+A delta 3D printer designed by 3 enthousiasts with cost and effectiveness.
+
+### History of the Gemstock
+
+In a 3D printing lab not so long ago,
+there was a man that who had in internship at that same lab.
```

Comment:
there was a man that who had in internship at that same lab.
there was a man that who had **an** internship at that same lab.